src/io/readers.ts

```diff
@@ -1,8 +1,9 @@
 import vtkITKImageReader from '@kitware/vtk.js/IO/Misc/ITKImageReader';
+import { convertItkToVtkImage } from '@kitware/vtk.js/Common/DataModel/ITKHelper';
 import { readImageArrayBuffer, extensionToImageIO } from 'itk-wasm';
+import { readImage } from '@itk-wasm/image-io';
 import { FileReaderMap } from '.';
 
-import { readFileAsArrayBuffer } from './io';
 import { stlReader, vtiReader, vtpReader } from './vtk/async';
```

Comment:
Looks like `itk-wasm/mesh-io` also has a similar generic reader for polydata? Maybe in the future we could get rid of these special readers as well.

Reply:
Yes, cool.  VolView's mesh support is just vestigial at the moment.